### ISSUE TITLE
SearchParameter

### ISSUE THREAD

**Issue by julsas** [medizininformatik-initiative/kerndatensatzmodul-onkologie#122]
Die SearchParameter für Core Ressourcen und Elemente sind jetzt in Meta: https://github.com/medizininformatik-initiative/kerndatensatz-meta/pull/88
Die Suchparamater auf Extensions sind so spezifisch, dass ich diese in dem Modul hier lassen würde.

Folgendes scheint außerdem noch fehlerhaft bei den Onko spezifischen SearchParams:

https://github.com/medizininformatik-initiative/kerndatensatzmodul-onkologie/blob/37cfc075502d2c220be86bfb7cea5e3394982285/input/fsh/searchParameter.fsh#L122

Die Extension URL scheint nicht zu stimmen.

---

https://github.com/medizininformatik-initiative/kerndatensatzmodul-onkologie/blob/37cfc075502d2c220be86bfb7cea5e3394982285/input/fsh/searchParameter.fsh#L259

Die Extension URL scheint nicht zu stimmen.

**Comment by ThomasDeBe** [comment]
ja, die wurden nochmal nach der Kommentierung refactored als stellungzurop, danke dir! 
ich korrigiere das

**Comment by julsas** [comment]
https://github.com/medizininformatik-initiative/kerndatensatz-meta/blob/58237fb1aa6d3b159c3b01c77bcd75606e3ee30d/input/fsh/searchparameters/mii-sp-meta-onko.fsh#L21

der Suchparameter aus Onko übernommen ist mir jetzt auch noch als fehlerhaft aufgefallen. Condition.evidence ist ein BackboneElement. Darauf kann man nicht mit type reference suchen. Condition.evidence.detail ist type reference. Dann muss das aber auch so in der expression stehen und SearchParameter.target spezifiziert werden, auf welchen Ressourcentyp als Ziel gesucht wird

**Comment by ThomasDeBe** [comment]
Ja, evidence-detail mit Ressource "List"

**Comment by ThomasDeBe** [comment]
Außerdem müssen wir bei AdverseEvent nochmal umprofilieren von AdverseEvent.severity auf AdverseEvent.seriousness, hier muss dann auch der Search Parameter angepasst werden.
Soll ich dir für die beiden Sachen (plus ggfs die dadrüber) einen PR stellen? Wenn ja gegen welchen Branch? @julsas

**Comment by julsas** [comment]
kannst einen PR gegen Master stellen
